merlin/dag/executors.py

```diff
@@ -512,7 +512,7 @@ def fit_phase(self, dataset, nodes, strict=False):
 
         dask_client = global_dask_client()
         if dask_client:
-            results = [r.result() for r in dask_client.compute(stats)]
+            results = [r.result() for r in dask_client.compute(stats) if r is not None]
```

Comment:
what's the scenario where the fit method of a on (Subraph op) returns None?

Reply:
Always, because it does not have stats itself. It has operators within it that have stats. So it will return the stats when its graph runs the fit but it wont bubble that up.

Reply:
> stats is empty and then there is not r

if stats is empty (`[]`) then the dask client compute also returns an empty list `dask_client.compute([]) => []`. and r is never reached as None in that case.